app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/DisplayImageSetActivityTest.kt

```diff
@@ -64,12 +64,14 @@ class DisplayImageSetActivityTest {
 
     @Test
     fun imageSetGridAndNameAreDisplayed() {
-        onView(withId(R.id.display_image_set_imagesGridView)).check(
-            ViewAssertions.matches(
-                ViewMatchers.isDisplayed()
+        if(dsPictures.isNotEmpty()) {

```

Comment:
Please use [assumeThat](https://junit.org/junit4/javadoc/4.12/org/junit/Assume.html) to indicate that a test cannot run.

Reply:
Oh yes Thank you for the tip